multiplayer/src/components/KeyboardControlsInfo.tsx

```diff
@@ -32,6 +34,14 @@ export default function Render() {
                     </div>
                 );
             })}
+            <div className="keymap-row tw-flex tw-flex-row tw-items-center">
+                <div className="keymap-key tw-align-middle tw-min-w-[2.5rem] tw-text-center tw-pb-[2.1rem]">
+                    1-6
+                </div>
+                <span className="keymap-name tw-font-semibold tw-text-center">
+                    <ReactionsIcon className="hover:tw-scale-125 tw-ease-linear tw-duration-[50ms]" />
```

Comment:
So does this actually work if they click on it?

Reply:
No, this one's just the icon that's used as a label for the button in `Reactions.tsx`, not the button itself (I did forget to save after removing the hover state though, so removed that~)